fapolicy_analyzer/ui/profiler_page.py

```diff
@@ -0,0 +1,114 @@
+# Copyright Concurrent Technologies Corporation 2021
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import glob
+import logging
+import gi
+import os
+
+from time import sleep
+from fapolicy_analyzer.ui.store import get_system_feature
+from fapolicy_analyzer.ui.ui_page import UIAction, UIPage
+from fapolicy_analyzer.ui.ui_widget import UIConnectedWidget
+from fapolicy_analyzer.ui.configs import Sizing
+from fapolicy_analyzer.ui.faprofiler import FaProfiler
+gi.require_version("Gtk", "3.0")
+from gi.repository import Gtk
```

Comment:
```suggestion
gi.require_version("Gtk", "3.0")
from gi.repository import Gtk   # isort: skip
```

Can you please add the isort comment? I use a plugin that sorts my imports and that prevents it from being sorted above the GTK version check line.